src/main/resources/io/relution/jenkins/awssqs/i18n/sqstriggerqueue/Messages.properties

```diff
@@ -6,3 +6,4 @@ warningUrl=Name or URL of an SQS queue is required
 errorUrlCodecommit=This is a CodeCommit URL, please provide a queue name or SQS URL
 errorUrlUnknown=This is not an SQS URL, please provide a queue name or SQS URL
 errorUuid=UUID is not unique
+errorMaxNumberOfJobQueue=Max. number of buildable items in the job queue must be a number between 0 and 100000
```

Comment:
is it OK to please move this line under `errorMaxNumberOfMessages` for order's sake ?

Reply:
👍 moved property